src/psm/src/ir_solver.cpp

```diff
@@ -452,14 +456,74 @@ void IRSolver::readC4Data(bool require_voltage)
                           + offset_x + centering_offset_x;
         const int y_cor = (bump_pitch_y_ * i) + offset_y + centering_offset_y;
         if (x_cor <= coreW && y_cor <= coreL) {
-          C4Bumps_.push_back(
-              {x_cor, y_cor, bump_size_ * unit_micron, supply_voltage_src});
+          sources_.push_back({x_cor,
+                              y_cor,
+                              bump_size_ * unit_micron,
+                              supply_voltage_src,
+                              true});
         }
       }
     }
   }
 }
 
+bool IRSolver::createBTerms(dbNet* net, double voltage)
```

Comment:
Confusingly named as it doesn't create bterms.  Perhaps createSourcesFromBTerms?